src/components/modules/mypage/MypageClient.tsx

```
@@ -31,7 +20,7 @@ const MypageClient = () => {
           <h1 className="text-base font-semibold text-gray-900 sm:text-2xl">마이페이지</h1>
 
           <button
-            onClick={handleModalOpen}
+            onClick={() => setIsModalOpen(true)}
             className="cursor-pointer lg:absolute lg:top-[68px] lg:right-[52px]"
           >
             <Image src="/icons/ic_mypage_edit.svg" alt="수정 아이콘" width={32} height={32} />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)